org-cyf/hugo.toml

```diff
@@ -1,5 +1,11 @@
 title = "CYF Curriculum"
 
+
+[taxonomies]
+  category = 'categories' # Content types are blocks, definitions, guides
+  tag = 'themes'
```

Comment:
Can we also add "thread" here so we can go through blocks in e.g. JS1 and annotate them with metadata e.g.
thread="Let's build a mental model for xxx"

Reply:
Yes we can - but look at Daniel's #530  (we can have both - there's no reason not to have multiple metadata slicings)